jwt-ballerina/jwt_validator.bal

```diff
@@ -523,12 +558,47 @@ isolated function validateAudience(Payload payload, string|string[] audienceConf
                 }
             }
         }
-        return prepareError("Invalid audience.");
+        return prepareError("JWT contained invalid audience.");
     } else {
         return prepareError("JWT must contain a valid audience.");
     }
 }
 
+isolated function validateJwtId(Payload payload, string jwtIdConfig) returns Error? {
+    string? jwtIdPayload = payload?.jti;
+    if (jwtIdPayload is string) {
+        if (jwtIdPayload != jwtIdConfig) {
+            return prepareError("JWT contained invalid JWT ID '" + jwtIdPayload + "'");
+        }
+    } else {
+        return prepareError("JWT must contain a valid JWT ID.");
+    }
+}
+
+isolated function validateKeyId(Header header, string keyIdConfig) returns Error? {
+    string? keyIdHeader = header?.kid;
+    if (keyIdHeader is string) {
+        if (keyIdHeader != keyIdConfig) {
+            return prepareError("JWT contained invalid key ID '" + keyIdHeader + "'");
+        }
```

Comment:
```suggestion
    if (keyIdHeader is string && keyIdHeader != keyIdConfig) {
        return prepareError("JWT contained invalid key ID '" + keyIdHeader + "'");
```

Reply:
@ayeshLK Here I want to simply ignore the following case.
```ballerina
keyIdHeader is string and keyIdHeader == keyIdConfig
```
With the suggested approach, the above case will definitely go to the else part. But there I want to handle `keyIdHeader is not string` separately. So, there need to be another `if` condition. WDYT?